docs/DeveloperGuide.adoc

```diff
@@ -852,6 +1033,16 @@ _{More to be added}_
 .  Should work on any <<mainstream-os,mainstream OS>> as long as it has Java `1.8.0_60` or higher installed.
 .  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
```

Comment:
As discussed in the tutorial, vague terms like "noticeable sluggishness", "average speed", etc., should be avoided in NFRs.